pkg/types/object_storage.go

```diff
@@ -27,5 +27,6 @@ type ObjectStorageInfo struct {
 
 type ObjectStoragePathDetail struct {
 	FilePath        string `bson:"file_path" json:"file_path" yaml:"file_path"`
+	AbsFilePath     string `bson:"abs_file_path" json:"-" yaml:"-"`
```

Comment:
If the key is "-",the filed is ignored,so are you sure the test is ok?

Reply:
This field will be omitted so this won't work

Reply:
I don't want to expose this to frontend so I set it as `-`. I used to test it by building a new reaper and cp to the resource container. It seems that 1.12 use a reaper image directly. I'll test with the new way in this weekend.